File: src/main/java/com/example/projetosisu/Curso.java
package com.example.projetosisu;

import java.util.ArrayList;
import java.util.List;

public class Curso {
    private String nome;
    private String campus;
    private List<Nota> notas;

    public Curso(String nome, String campus) {
        this.nome = nome;
        this.campus = campus;
        this.notas = new ArrayList<>();
    }

    public String getNome() { return nome; }
    public String getCampus() { return campus; }
    public List<Nota> getNotas() { return notas; }

    public void adicionarNota(Nota nota) {
        this.notas.add(nota);
    }
}

File: src/main/python/sisu_analyser/Curso.py
from typing import List
from .Nota import Nota

class Curso:
    def __init__(self, nome: str, campus: str):
        self.__nome = nome
        self.__campus = campus
        self.__notas: List[Nota] = []
        
    def get_nome(self):
        return self.__nome
    
    def get_campus(self):
        return self.__campus
    
    def get_notas(self):
        return self.__notas
        
    def adicionar_nota(self, nota: Nota):
        self.__notas.append(nota)